tests/functional/README.md

```diff
@@ -1,18 +1,38 @@
 Running the Functional Tests
-==
+============================
+
+Functional tests run against a set of Docker containers that mimic the AWS stack this service runs on
+in production.
+
+To run the functional tests with a Python 2.7 image, do this:
 
 ```bash
-> docker build -t fsm -f tests/functional/Dockerfile .
-> docker build -t fsm_docker_runner -f tools/experimental/Dockerfile.fsm_docker_runner .
-> docker build -t dev_ecs -f tools/experimental/Dockerfile.dev_ecs .
-> docker run \
+$ docker build -t fsm_docker_runner -f tools/experimental/Dockerfile.fsm_docker_runner .
```

Comment:
`Dockerfile.fsm_docker_runner` and `Dockerfile.dev_ecs` need py3 versions too

Reply:
`Dockerfile.fsm_docker_runner` seems to have disappeared in this branch

Reply:
OK, so Python 3 compatibility for the *production* code is the goal here.

I'm not sure why experimental tools with independent Dockerfiles *need* both Python 2 & 3 Dockerfiles. It seems that it would suit our needs to stick with one or the other.

Reply:
> Dockerfile.fsm_docker_runner seems to have disappeared in this branch

It still shows for me, and I don't see removal of that file in the changelog.

Reply:
i see them now. i forgot to pull. you still need python3 version. might be quicker if I just make those commits.

Reply:
Let me explain where I'm coming from regarding Python 3 versions of those Dockerfiles.

So the need here is for a version of the FSM library that is compatible with both Python 2.7 and 3.6+. That's the goal. It made sense to create a Python 3 version of the main Dockerfile, *even though* it unfortunately complicates the functional testing instructions, because it allows for functional testing of the code base with Python 3, and catches issues that otherwise would not be caught.

These other Dockerfiles, however, look like independent tools that serve the role of mocking infrastructure. They could be written in Python 2, Python 3, Java, Clojure... it doesn't really matter to the end user, thanks to Docker. To maintain both Python 2 & Python 3 versions of these would further complicate the functional testing story with no tangible benefit.

Reply:
Thank you for making a py3 PR to this open source repo. I intend to merge it once it meets my standards and conventions, both documented and undocumented. I will strive to make the documentation better for future contributors. I am the maintainer, evangelist, caretaker, and owner of this repo. Before any PR gets merged, I need to ensure it does not create immediate tech debt for myself as I build out new features. 

I have made the PR already https://github.com/aaronkavlie-wf/aws-lambda-fsm-workflows/pull/1

Reply:
Despite my reservations, I just merged in that PR.

Aside from some conflicts that just cropped up, the only outstanding item now is the kinesalite version confusion above.